common/am/lib/src/main/java/org/apache/syncope/common/lib/auth/SyncopeAuthModuleConf.java

```diff
@@ -55,6 +57,10 @@ public void setCredentialCriteria(final String credentialCriteria) {
         this.credentialCriteria = credentialCriteria;
     }
 
+    public int getMaxRetryAttempts() { return maxRetryAttempts; }
+
+    public void setMaxRetryAttempts(int maxRetryAttempts) { this.maxRetryAttempts = maxRetryAttempts; }
```

Comment:
this is not going to pass checkstyle: ensure to run the build locally before submitting